workshops/util.py

```diff
@@ -315,3 +318,79 @@ def merge_persons(person_from, person_to):
 
     # removes tasks, awards, qualifications in a cascading way
     person_from.delete()
+
+
+class WrongEventURL(Exception):
+    pass
+
+
+def normalize_event_index_url(url):
```

Comment:
This would be a significant difficulty. The current code reaches to `raw.githubcontent.com` for the index file of the workshop. How would we read the metadata if the index file is not accessible?

One solution would be to add all necessary variables to the `<meta>` of the generated `index.html` (this solution requires changes to the `workshop-template`). But what about historical workshops? We'd have to support both versions…